src/main/resources/swagger.api/schemas/centralServerSettingsDTO.json

```diff
@@ -0,0 +1,20 @@
+{
+  "description": "Central server configuration",
+  "type": "object",
+  "properties": {
+    "centralServerId": {
+      "description": "Central server id",
+      "type": "string",
+      "format": "UUID"
+    },
+    "checkPickupLocation": {
+      "description": "Check for pickup location",
```

Comment:
Just use the definition from story.
indicate whether to look up pickup locations for INN-Reach item hold requests based on transaction pickupLocation

Reply:
replaced